pandas/core/categorical.py

```diff
@@ -1157,30 +1157,74 @@ def argsort(self, ascending=True, **kwargs):
         return result
 
     def sort_values(self, inplace=False, ascending=True, na_position='last'):
-        """ Sorts the Category by category value returning a new Categorical by
-        default.
-
-        Only ordered Categoricals can be sorted!
-
-        Categorical.sort is the equivalent but sorts the Categorical inplace.
+        """ Sorts the Categorical by category value returning a new
+        Categorical by default. While an ordering is applied to the
```

Comment:
Can you leave the first sentence alone (apart from the Category -> Categorical correction of course). I mean, leave a blank line before the rest of the paragraph